tests/test_Admin.py

```diff
@@ -320,6 +320,41 @@ def test_alter_configs_api():
             f.result(timeout=1)
 
 
+def test_incremental_alter_configs_api():
+    a = AdminClient({"socket.timeout.ms": 10})
+
+    with pytest.raises(TypeError):
+        a.incremental_alter_configs(None)
+
+    with pytest.raises(ValueError):
```

Comment:
This should be `TypeError`

Reply:
string is interpreted as iterable and _make_futures raises
`raise ValueError("Expected list of {}".format(repr(class_check)))`
changing it would be breaking for the users.
I could pass the error to raise as parameter for the legacy functions by changing `_make_futures` for newer methods

Reply:
Shall we add a new `_make_futures` for this use case where we raise the correct error and add a FIXME to fix Error in major update in old APIs?